docs/application/vstools/install.md

```diff
@@ -17,8 +17,8 @@ To work with Visual Studio Tools for Tizen, your computer must have:
 
 - At least 1.5 GB of available disk space
 - Visual Studio 2017 to use Tizen 4.0 and 5.0
-- Visual Studio 2019 to use Tizen 4.0 and higher
-- Visual Studio 2022 to use Tizen 6.5 and higher
+- Visual Studio 2019 to use Tizen 4.0 and 6.5
+- Visual Studio 2022 to use Tizen 4.0 and higher
 - Latest Tizen Tools updates support Tizen Native and Web app creation and is provided with Tizen Studio version 4.5 and above. Make sure the same is installed or [updated](../tizen-studio/setup/update-sdk.md) through Tizen package manager.
```

Comment:
An article should be used before the word “latest” and the correct verb to be used here is “are”. 
```suggestion
- The latest Tizen Tools updates support Tizen Native and Web app creation and are provided with Tizen Studio version 4.5 and above. Make sure the same is installed or [updated](../tizen-studio/setup/update-sdk.md) through Tizen package manager.
```

Reply:
@sangwook96 this comment is not updated as well.

Reply:
@sangwook96 perhaps you missed this comment. Please incorporate this change. Thanks. 
```suggestion
- The latest Tizen Tools updates support Tizen Native and Web app creation and are provided with Tizen Studio version 4.5 and above. Make sure the same is installed or [updated](../tizen-studio/setup/update-sdk.md) through Tizen package manager.
```